app/build.gradle

```diff
@@ -101,15 +97,15 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
 
     // Miscellaneous
-    implementation 'com.google.guava:guava:28.2-android'
+    implementation 'com.google.guava:guava:29.0-android'
     implementation 'com.annimon:stream:1.2.1'
-    implementation 'com.google.android:flexbox:1.1.0'   // TODO To remove with the old home screen
-    implementation 'com.squareup.okhttp3:okhttp:3.14.6'
-    implementation 'org.sufficientlysecure:html-textview:3.4'
+    implementation 'com.google.android:flexbox:1.1.1'   // TODO To remove with the old home screen
```

Comment:
I need to remove it...

Reply:
Yeah, I know. I'm just updating it to the latest non breaking for now until you replace it.